Tests/QuickTests/QuickFocusedTests/FocusedTests.swift

```diff
@@ -63,6 +63,6 @@ final class FocusedTests: XCTestCase, XCTestCaseProvider {
             _FunctionalTests_FocusedSpec_Focused.self,
             _FunctionalTests_FocusedSpec_Unfocused.self
         ])
-        XCTAssertEqual(result?.executionCount, 8 as UInt)
+        XCTAssertEqual(result?.executionCount, 8)
```

Comment:
`executionCount` is `Int` in Swift 3.2 / Swift 4.

Reply:
Do the tests still compile in Swift 3.1? Or was this redundant?

Reply:
See the build matrix, this does still compile in Swift 3.1. In that case `8` should be inferred as `UInt` I think.